python/poetry.lock

```diff
@@ -99,7 +99,7 @@ tests_no_zope = ["cloudpickle", "coverage[toml] (>=5.0.2)", "hypothesis", "mypy
 
 [[package]]
 name = "autoflake"
-version = "1.6.0"
+version = "1.6.1"
```

Comment:
Why is a poetry.lock update in this PR? Do you need this? There are no updates to the install time dependencies in pyproject.toml